README.md

```diff
@@ -257,6 +257,8 @@ If you use Kafka broker 0.9 or 0.8 you must set
 `broker.version.fallback` to your broker version,
 e.g `broker.version.fallback=0.9.0.1`.
 
+Please note that brokers prior to version 0.9 does not support `Consumer.subscribe()`. If your server is version 0.8.x, please use [`Consumer.assign()`](https://docs.confluent.io/current/clients/confluent-kafka-python/#confluent_kafka.Consumer.assign) instead.
```

Comment:
This list of unsupported APIs in earlier broker versions will grow long. I'd prefer if this was just mentioned in each method's doc string.

The client should also raise an error (if an error_cb is set) when this happens, did it?

Reply:
yeah, I agree

It did raise an error. I don't remember which on exactly. 

It think it said that JoinGroup was not supported and had a hint suggesting that the `api.version.request` was not set to `false` or `broker.version.fallback` was not set to the correct value